src/gluonts/mx/trainer/_base.py

```diff
@@ -122,10 +127,7 @@ def __init__(
         weight_decay: float = 1e-8,
         init: Union[str, mx.initializer.Initializer] = "xavier",
         hybridize: bool = True,
-        avg_strategy: Union[
-            AveragingStrategy, IterationAveragingStrategy
-        ] = SelectNBestMean(num_models=1),
-        post_initialize_cb: Optional[Callable[[mx.gluon.Block], None]] = None,
+        callbacks: Optional[List[Callback]] = None,
```

Comment:
Good point.  I changed that. However, this makes this change breaking, since some parameter of the trainer are not used anymore (e.g. learning_rate_decay_factor)...